request_a_govuk_domain/request/utils.py

```diff
@@ -1,17 +1,33 @@
 import logging
 import os
 import uuid
-from typing import List
+from typing import List, Dict
```

Comment:
Since this is python >= 3.8, I don't think you need this and can use the lower case versions of `list`, `dict` etc without importing them.
https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html

Reply:
Done